stylesheet/main.css

```diff
@@ -200,44 +200,43 @@ nav{
 
 
 .glossary{
-    column-span:all;
+    grid-column: span 2;
     
 }
-.useful-links{
-    column-span: all;
+#useful-links{
+    grid-column: span 2;
 }
 
 body{
-    height: 100%;
-    min-width: 999px;
+    min-height: 100vh;

```

Comment:
No need for a min-width property here
Use vh instead of % to make a height relative to the viewport height